styles/style.css

```diff
@@ -235,12 +226,46 @@ html {
   overflow: hidden;
 }
 
+html {
+  font-size: 125%;
+  color: #fff;
+  text-rendering: optimizeLegibility;
+}
+
+h1, h2, h3, p {
+  margin: 0;
+  padding: 0;
+}
+
+h1 {
+  font-family: Neonderthaw, cursive;
+  font-size: clamp(4rem, 30vw, 8rem);
+}
+
+footer span {
+  font-family: "Permanent Market", cursive;
+}
+
 body {
+  position: relative;
   min-height: 100vh;
   display: grid;
   grid-template-rows: auto 1fr auto;
   grid-template-columns: 100%;
 }
+body::before {
```

Comment:
I think if we use linear-gradient on background image, this can be voided.

Reply:
I'll have to try that.